tensorflow/core/distributed_runtime/eager/eager_service_impl.cc

```diff
@@ -80,6 +80,9 @@ Status GetNumRetvals(tensorflow::EagerContext* context, const string& op_name,
 
 Status EagerServiceImpl::CreateContext(const CreateContextRequest* request,
                                        CreateContextResponse* response) {
+  if (env_ == nullptr || env_->rendezvous_mgr == nullptr) {
+    return errors::InvalidArgument("invalid eager env_ or env_->rendezvous_mgr.");
```

Comment:
I think this should be an errors::Internal failure rather than an InvalidArgument failure.

Reply:
right,  error::internal failure was  suitable 。